### PR TITLE
Prevent insertion of font links in head element

### PR DESCRIPTION
**Pull request by blamens**
Added check on ENV.useFontsFromAssets to prevent the insertion of links to fonts.googleapi.com in the contentFor hook

### CODE REVIEW

**Comment by blamens** [comment]
- Renamed config flag to `insertFontLinks` and added it to `ember-paper` hash in ENV
- Used the config hook to set the default of the flag
- Added entry to changelog

Let me know if I need to change anything else!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Version 1.0 introduces many API changes relative to previous releases. In additi
 
 Contributions and pull requests are always welcome. Contributors may often be found on the [#e-paper channel on slack](https://embercommunity.slack.com/messages/e-paper/). Building the dummy application by installing `ember-paper` as if it were an application will provide you an up-to-date interactive demo, templates, and code samples.
 
+- Consuming apps can now specify `ENV['ember-paper'].insertFontLinks` to prevent the insertion of fonts.googleapi links in the head tag
```
Comment (miguelcobain) [inline CHANGELOG.md]: The entry should be on a `### master` version below.

File: index.js [hunk 2]
```diff
@@ -37,10 +37,16 @@ module.exports = {
...
 
+  config(env, baseConfig) {
+    return { 'ember-paper': { insertFontLinks: true } };
+  },
+
   contentFor: function(type, config) {
     if (type === 'head') {
-      return '<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">' +
-        '<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">';
+      if (config['ember-paper'].insertFontLinks) {
```
Comment (miguelcobain) [inline index.js]: You still need to test if `type === 'head'`, right?

**Comment by blamens** [comment]
The nested `if` blocks are intentional (if some other stuff needs to be done to the `head` element in the future).

I'm not sure I understand the changelog structure. Should the entry be placed like this? In the `1.0.0-alpha.15 (January 26, 2017)` section?

**Comment by anthonycollini** [comment]
Can we add this option to the documentation as well?

**Comment by miguelcobain** [comment]
@anthonycollini done. Will appear when the demo app is deployed again.

**Comment by v3ss0n** [comment]
Thanks , thats very good!

**Pull request merged**